plugins/inventory/now.py

```diff
@@ -0,0 +1,443 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import os
+
+from ansible.errors import AnsibleParserError
+from ansible.plugins.inventory import BaseInventoryPlugin, to_safe_group_name
+
+from ..module_utils.client import Client
+from ..module_utils.table import TableClient
+
+
+DOCUMENTATION = r"""
```

Comment:
Nitpick: the indentation of the documentation block is a mix of 2 and 4 spaces. We should pick one and stick to it. Maybe 2 is the right answer here since we already use this in all other modules?